docs/wxm-devices/helium-bundle/helium-troubleshooting.mdx

```diff
@@ -35,7 +27,7 @@ To factory reset, hold down the middle button on the bottom of the weather stati
 Restart your station. Pairing is only allowed in the first 5 minutes after the station is started. After 5 minutes the station refuses pairing. If the problem persists, do a factory reset and try again. 
 
 :::note
-The Helium station is reset to factory settings by pressing and holding the reset button for ~5 seconds. When released, a pulsating red indicator should be displayed to the right of it. This indicates a successful reset to the factory settings of the helium station.
+The Helium station is reset to factory settings by pressing and holding the reset button for ~10 seconds. When released, a pulsating red indicator should be displayed to the right of it. This indicates a successful reset to the factory settings of the helium station.
 <img src={useBaseUrl("img/wxm-devices/helium-bundle/reset_station.svg")} />
 :::
```

Comment:
Remove this note and hyperlink in the text above "do a factory reset" the new "Factory Reset" section you created below.